CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.4.7 (2020-05-07)
+
+- Fixed the colorpicker demo to no longer have redundant examples. [#44](https://github.com/blackbaud/skyux2-demos/pull/44)
```

Comment:
How about "remove" instead of "to no longer have"?

Reply:
Works for me. Fixed!